src/paas_app_charmer/fastapi/charm.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+"""Deprecated FastAPI Charm service."""
+
+import paas_charm.fastapi.charm
+
+
+class Charm(paas_charm.fastapi.charm.Charm):
+    """Deprecated FastAPI charm."""
```

Comment:
Perhaps we include a note here that these will be maintained for the life of 24.04 and 22.04? For 26.04 they would be removed

Reply:
For the other deprecated classes as well

Reply:
Sure. Let's be explicit about that.

The real dependency is with the major version 1. Old projects (from 22.04 and 24.04) that do `charmcraft pack` will get the latest 1.X.X version of this project, so the way of breaking the dependency is changing the major version to 2 for 26.04 bases (and state it that way in charmcraft). 

If the major is not updated when bases 26.04 are in use, I think we will have to maintain the deprecated classes (and the backward compatibility). When 26.04 is added, if we release version 2, we will have to maintain two different branches.